core/domain/services/admin/ExitModal.php

```diff
@@ -2,6 +2,7 @@
 namespace EventEspresso\core\domain\services\admin;
 
 use EventEspresso\core\services\assets\Registry;
+use const FILTER_VALIDATE_BOOLEAN;
```

Comment:
This will cause fatals  for PHP versions < 5.6 ... it's been removed multiple times, I wonder how it crept back in :(

Reply:
removed